nixos/modules/services/continuous-integration/buildkite-agent.nix

```diff
@@ -204,38 +204,39 @@ in
         environment = config.networking.proxy.envVars // {
           HOME = cfg.dataDir;
           NIX_REMOTE = "daemon";
+          BUILDKITE_SHELL = cfg.shell;
```

Comment:
Why not to add `shell` configuration option to the configuration file instead?

Reply:
this was cherry-picked from the older PR, I made it part of the configuration file now.